relay-sampling/src/dsc.rs

```diff
@@ -245,15 +246,15 @@ mod tests {
     fn parse_user() {
         let jsons = [
             r#"{
-                "trace_id": "00000000-0000-0000-0000-000000000000",
+                "trace_id": "67e5504410b1426f9247bb680e5fe0c8",
```

Comment:
Haven't taken a full look, so maybe it's there already, it would be good if we keep at least one test with the old format.

Reply:
A trace id is not valid if it's made up of only zeros: https://www.w3.org/TR/trace-context/#trace-id.

Reply:
I will keep some tests with the `-`.